### PR TITLE
Add Group Vibe Frontend

### PR DESCRIPTION
**Pull request by farooqashar**
This handles issue #16 

This PR does the following: 
- Sets up the two emojis for confused and happy on the sidebar of main messaging view that scale in size depending on counts
- Sets up socket events for sending over confused and happy votes
- Initially gets the group vibe counts when one joins a room
- Updates the local group vibe counts when there is a vote received by a user

### CODE REVIEW

File: frontend/src/views/MainChatView.vue [hunk 1]
```diff
@@ -16,8 +16,81 @@
...
+                v-if="this.confusedCount === 1"
+                src="https://img.icons8.com/ios-glyphs/35/null/question--v1.png"
+              />
+              <img
+                v-if="this.confusedCount === 2"
+                src="https://img.icons8.com/ios-glyphs/40/null/question--v1.png"
+              />
+              <img
+                v-if="this.confusedCount === 3"
+                src="https://img.icons8.com/ios-glyphs/50/null/question--v1.png"
```
Comment (Lavrans) [inline frontend/src/views/MainChatView.vue:41]: I think we should do this using some basic math and CSS properties

File: backend/api/index.ts [hunk 2]
```diff
@@ -134,6 +134,30 @@ io.on("connection", (socket) => {
...
+    );
+    socket.to(data.roomName).emit("confusedVote:received", totalCounts);
+  });
+
+  socket.on("happyVote", async (data) => {
+    const GroupVibeCollectionNOTREAL = "Is a Placeholder Collection"
+    const updateOneNOTREAL = "Is A Placeholder Function Within Collection"
+    // updateOneNOTREAL should take in a reaction, username, and chatroomKey and update the counts for the emojis and handle one user voting and handling toggling. In the end, it should return a json object like {happy: #count, confused: #count}
+    const totalCounts = await GroupVibeCollectionNOTREAL.updateOneNOTREAL(
+      data.reaction,
```
Comment (Lavrans) [inline backend/api/index.ts:154]: I think we should wait till backend collections are ready and implement this properly before we merge with main.

**Review by Lavrans — CHANGES_REQUESTED** [review]
I've made one comment on the front-end code. Otherwise I think we should wait until #45  is merged and implement socket backend to utilize real collections.

**Review by jmgcov — APPROVED** [review]
Lgtm

**Review by jmgcov — APPROVED** [review]
Ok!

**Pull request merged**